src/renderer/src/routes/pages/VoteNeo3/VoteNeo3List.tsx

```diff
@@ -0,0 +1,157 @@
+import { useMemo, useRef, useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { TbAlertTriangleFilled } from 'react-icons/tb'
+import { GetCandidatesToVoteResponse, GetVoteDetailsByAddressResponse } from '@cityofzion/bs-neo3/dist/interfaces'
+import { Tooltip } from '@renderer/components/Tooltip'
+import { VOTE_NEO3_COZ_PUB_KEY } from '@renderer/constants/public-keys'
+import { useMount } from '@renderer/hooks/useMount'
+import { IAccountState } from '@shared/@types/store'
+import { UseQueryResult } from '@tanstack/react-query'
+import { match, P } from 'ts-pattern'
+
+import { VoteNeo3ListItem } from './VoteNeo3ListItem'
+import { VoteNeo3NotFound } from './VoteNeo3NotFound'
+import { VoteNeo3Skeleton } from './VoteNeo3Skeleton'
+
+type TProps = {
+  neo3Account?: IAccountState
+  voteErrorMessage: string
+  canVote: boolean
+  candidatesToVoteQuery: UseQueryResult<GetCandidatesToVoteResponse>
+  voteDetailsByAddressQuery: UseQueryResult<GetVoteDetailsByAddressResponse>
```

Comment:
You don't need to receive these prop as you already have a custom hook for them and could call these hooks inside here.

Take a look at this issue: https://github.com/TanStack/query/discussions/463